fhttp/httprunner.go

```diff
@@ -49,6 +49,51 @@ type HTTPRunnerResults struct {
 	aborter *periodic.Aborter
 }
 
+// Merge two different HTTPRunnerResults
+func Merge(rr1 *HTTPRunnerResults,
```

Comment:
it's great you have tests for the lower level (stats) one but this one also needs tests